client/install_cli_ubuntu.sh

```diff
@@ -308,7 +313,8 @@ setup_ipmi() {
   # check if ipmi is supported
   if ls /dev/ipmi* 1> /dev/null 2>&1; then
     echo "Installing FreeIPMI tool suite ..."
-    sudo apt-get update && sudo apt-get install freeipmi-tools -y --no-install-recommends && sudo rm -rf /var/lib/apt/lists/*
+    ($vvalue && sudo apt-get install freeipmi-tools -y --no-install-recommends && sudo rm -rf /var/lib/apt/lists/*) \
```

Comment:
Do we want the `-f` in `sudo rm` or should we not force remove?

Reply:
That is a good point. Let me check if we do actually need this command to run.

Not sure, why would we have added it.